test/CMakeLists.txt

```diff
@@ -62,18 +68,29 @@ endif()
 
 if(SINGULARITY_TEST_SESAME)
   target_compile_definitions(eos_tabulated_unit_tests PRIVATE SINGULARITY_TEST_SESAME)
+  target_compile_definitions(closure_unit_tests PRIVATE SINGULARITY_TEST_SESAME)
 endif()
 if(SINGULARITY_TEST_STELLAR_COLLAPSE)
   target_compile_definitions(eos_tabulated_unit_tests
                              PRIVATE SINGULARITY_TEST_STELLAR_COLLAPSE)
 endif()
+if(SINGULARITY_USE_SPINER)
+  target_compile_definitions(closure_unit_tests
```

Comment:
Are these preprocessor defines specific to this test or are they defined elsewhere as well @jhp-lanl ?

Reply:
I went ahead and switched to `SINGULARITY_USE_SPINER_WITH_HDF5`. Let me know if you're satisfied here.

Reply:
Going to resolve this

Reply:
did you remove this compile definition then as it seems to still be showing up in the cmake diff.

Reply:
Did I remove it? No. I didn't think it needed to be removed from your previous comment. This test shouldn't be run when `SINGULARITY_USE_SPINER_WITH_HDF5` is off.

I did forget to switch it to `SINGULARITY_USE_SPINER_WITH_HDF5` though. That has been corrected.

Reply:
The following macro should already be defined https://github.com/lanl/singularity-eos/blob/main/cmake/singularity-eos/hdf5.cmake#L92C21-L92C67 `SINGULARITY_USE_HDF5`

Reply:
I prefer using the `SINGULARITY_USE_SPINER_WITH_HDF5` flag since I'm specifically testing for spiner and HDF5 not just the latter.